addons/mail/static/src/views/web/activity/activity_cell.js

```diff
@@ -38,6 +42,14 @@ export class ActivityCell extends Component {
     get reportingDateFormatted() {
         return formatDate(luxon.DateTime.fromISO(this.props.reportingDate));
     }
+    get displayedSummaries() {
+        const summariesWithContent = this.props.summaries.filter((el) => !!el);
```

Comment:
"el" usually refers to a DOM element, here it's just a string right? Maybe "textContent" instead?